packages/ds-healthcare-gov/src/styles/components/_Inset.scss

```diff
@@ -1,17 +1,15 @@
-$hc-c-inset-border-width: $spacer-half;
-
 .hc-c-inset {
-  border-left: $hc-c-inset-border-width solid $color-primary;
+  border-left: $inset__border-width solid $color-primary;
   margin-bottom: $spacer-2;
   padding: $spacer-2;
 }
 
 // Align the inset's border with the checkbox/radio field when used as part
 // of the Exposed Within pattern
```

Comment:
[Question] It looks like there are situations where this component is used within choice fields. There is no example of this on the current HcGov doc site but wondering 1) If we should make a Story for this use-case & 2) if that should be done as part of this ticket or for future documentation updates?

Reply:
I think if we were to add a story for it, it would be safe to leave that for a future update. We probably want to look at that use case critically and determine if it's something we should consider deprecating. @line47, thoughts?